### ISSUE TITLE
No documentation for my code. Task 2.2 Gen_ai lab

### ISSUE THREAD

**Issue by J-Doza** [onasito/Group-5#96]
Some code i used for my machine learning class didnt have any documentation and was a little difficult to follow. 

`
import pandas as pd
import matplotlib.pyplot as plt
from sklearn.decomposition import PCA

data = pd.read_csv('MNIST_100.csv')

y = data.iloc[:, 0]
x = data.drop('label', axis=1)

pca = PCA(n_components=2)  # this function reduces to 2 dimensions
pca.fit(x)
PCAX = pca.transform(x)

fig, ax = plt.subplots()

size_of_data = len(PCAX)
size_of_class = 100

for i in range(0, size_of_data, size_of_class):
    ax.scatter(PCAX[i:i+size_of_class, 0], PCAX[i:i+size_of_class, 1],
               label=i//size_of_class)

ax.set_title("Task 1: Scatter plot of PCA")
ax.set_xlabel("PC1")
ax.set_ylabel("PC2")
ax.legend(loc='best', title='Classes', fontsize='xx-small')

plt.show()
`

Im going to ask chatgpt if it could write clear and concise documentation for me.

**Comment by J-Doza** [comment]
Pull request #97 closes this with a link to chatgpt recommendation
